### PR TITLE
MTM-42115-Treatment-of-Unicode-word-characters-in-passwords

### PR DESCRIPTION
**Pull request by Airdroplet**
* provided more precise description for allowed characters in a password acceptable by the platform

image

### CODE REVIEW

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 1]
```diff
@@ -117,7 +117,11 @@ Make sure to select a strong password. To support you in doing so, a password st
 By default, the password must meet the following conditions:
 
 * Consist of at least eight characters (this parameter can be configured by the {{< management-tenant >}}, see [Administration > Platform configuration settings](/users-guide/administration/#platform-configuration-settings).)
-* Include each of the following character types: uppercase letters, lowercase letters, numbers and symbols.
+* Include each of the following character types:
+  * uppercase letters: `[A-Z]`, for example `ABCDEF`.
+  * lowercase letters: `[a-z]`, for example `abcdef`.
+  * numbers: `[0-9]`, for example: `123456`.
+  * platform accepts any other symbol as special character: `[^0-9a-zA-Z]`, for example `!@#$%^`.
```
Comment (rssag) [inline content/users-guide/getting-started-bundle/user-settings.md:124]:
```suggestion
  * the platform accepts other characters as a special characters: `[^0-9a-zA-Z]`, for example `!@#$%^`.
```

Comment (Airdroplet) [reply content/users-guide/getting-started-bundle/user-settings.md:124]: I would rather stick to `symbols` here. Like: `the platform accepts other symbols as special characters`. This includes japanese, chinese, russian, etc... and also symbols like `;`.

Comment (BeateRixen) [reply content/users-guide/getting-started-bundle/user-settings.md:124]: Why do we have this phrasing "the platform accepts ...". Couldn´t we simply say "any other symbol as special characters ..."

Comment (Airdroplet) [reply content/users-guide/getting-started-bundle/user-settings.md:124]: actually I like it. Will do the chage suggested by Beate.

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 2]
```diff
@@ -117,7 +117,11 @@ Make sure to select a strong password. To support you in doing so, a password st
 By default, the password must meet the following conditions:
 
 * Consist of at least eight characters (this parameter can be configured by the {{< management-tenant >}}, see [Administration > Platform configuration settings](/users-guide/administration/#platform-configuration-settings).)
-* Include each of the following character types: uppercase letters, lowercase letters, numbers and symbols.
+* Include each of the following character types:
+  * uppercase letters: `[A-Z]`, for example `ABCDEF`.
+  * lowercase letters: `[a-z]`, for example `abcdef`.
+  * numbers: `[0-9]`, for example: `123456`.
+  * any other symbol as special character: `[^0-9a-zA-Z]`, for example `!@#$%^`.
```
Comment (darluk) [inline content/users-guide/getting-started-bundle/user-settings.md:124]: as special => `as a special`

**Review by darluk — APPROVED** [review]

**Review by darluk — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 3]
```diff
@@ -117,7 +117,11 @@ Make sure to select a strong password. To support you in doing so, a password st
 By default, the password must meet the following conditions:
 
 * Consist of at least eight characters (this parameter can be configured by the {{< management-tenant >}}, see [Administration > Platform configuration settings](/users-guide/administration/#platform-configuration-settings).)
-* Include each of the following character types: uppercase letters, lowercase letters, numbers and symbols.
+* Include each of the following character types:
```
Comment (pawel-rynarzewski-sag) [inline content/users-guide/getting-started-bundle/user-settings.md:120]: "each" only applies to having strong/green passwords. If we talk about passwords in general, it's not required to have character from each of these groups. Recommended - sure, but this is supposed to be a list of conditions that **must** be met.

Comment (Airdroplet) [reply content/users-guide/getting-started-bundle/user-settings.md:120]: We're talking about default setting for password here. By default `green password` is enabled.

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 4]
```diff
@@ -117,7 +117,11 @@ Make sure to select a strong password. To support you in doing so, a password st
...
+  * uppercase letters: `[A-Z]`, for example `ABCDEF`.
+  * lowercase letters: `[a-z]`, for example `abcdef`.
+  * numbers: `[0-9]`, for example: `123456`.
+  * any other symbol as a special character: `[^0-9a-zA-Z]`, for example `!@#$%^`.
```
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/getting-started-bundle/user-settings.md]: you right, listed all allowed.

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 5]
```diff
@@ -117,7 +117,11 @@ Make sure to select a strong password. To support you in doing so, a password st
 By default, the password must meet the following conditions:
 
 * Consist of at least eight characters (this parameter can be configured by the {{< management-tenant >}}, see [Administration > Platform configuration settings](/users-guide/administration/#platform-configuration-settings).)
```
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/getting-started-bundle/user-settings.md:119]: there is a reference to setting description, don't see a need to mention it here again.

File: content/users-guide/getting-started-bundle/user-settings.md (hunk 3 above)
Comment (pawel-rynarzewski-sag) [reply content/users-guide/getting-started-bundle/user-settings.md:120]: then ok :+1:

**Pull request merged**